content/docs/getting_started/traffic_split.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "OSM manual demo"
+description: "The manual demo is a step-by-step walkthrough set of instruction of the automated demo."
+type: docs
+weight: 4
+aliases: = ["/docs/install/manual_demo/"]
+---
+
+### Configure Traffic Split between Two Services
+
+We will now demonstrate how to balance traffic between two Kubernetes services, commonly known as a traffic split. We will be splitting the traffic directed to the root `bookstore` service between the backend services `bookstore-v1` and `bookstore-v2`.
+
+### Deploy bookstore v2 application
+
+To demonstrate usage of SMI traffic access and split policies, we will now deploy version v2 of the bookstore application (`bookstore-v2`) - remember that if you are using openshift, you must add the security context constraint to the bookstore-v2 service account as specified in the [installation guide](/docs/install/#openshift). A `bookstore` service will also be created to serve as the root of our traffic split between `bookstore-v1` and `bookstore-v2`.
+
+```bash
+# Contains the bookstore-v2 Kubernetes Service, Service Account, Deployment and SMI Traffic Target resource to allow
+# bookbuyer to communicate with `bookstore-v2` pods
+kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v0.9/docs/example/manifests/apps/bookstore-v2.yaml
+```
+
+Next, update `bookbuyer` and `bookthief` to send requests to the `bookstore` service instead of the `bookstore-v1` service. This will restart each deployment and reset the displayed counts of books bought to 0:
+
+```bash
+kubectl set env -n bookbuyer deployment/bookbuyer -c bookbuyer BOOKSTORE_SVC=bookstore
+kubectl set env -n bookthief deployment/bookthief -c bookthief BOOKSTORE_SVC=bookstore
+```
+
+Wait for the `bookstore-v2` pod to be running in the `bookstore` namespace. Next, exit and restart the `./scripts/port-forward-all.sh` script in order to access v2 of bookstore.
+
+- [http://localhost:8082](http://localhost:8082) - **bookstore-v2**
+
+The counter should _not_ be incrementing because no traffic is flowing yet to the `bookstore-v2` service.
```

Comment:
remove "yet". No traffic will ever flow to the `bookstore-v2` service, but rather the `bookstore` service.